backend/backend/settings.py

```diff
@@ -110,6 +111,12 @@
 MEDIA_URL = 'media/'
 MEDIA_ROOT = BASE_DIR / MEDIA_URL
 
+# Эти настройки используются для отладки, пока нет настройки сервера отправки писем
+# Пользователь дёргает ручку /auth/users/reset_password/ ->>
+# в папку sent_email отправляется письмо, где содержится ссылка для сброса пароля
+EMAIL_BACKEND = 'django.core.mail.backends.filebased.EmailBackend'
```

Comment:
Макс, сейчас приходит кастомное письмо от djoser, это ок.
Попробуй кастомизировать текст письма, чтобы быть готовым быстро внести соответствующую правку в будущем, когда появится корретное содержимое. Добавь туда тот текст, какой видишь сам, исходя из бизнес-логики сервиса (мы восстанавливаем доступ представителю больницы, как это можно было бы +- оформить?)

```txt
Вы получили это письмо, потому что вы (или кто-то другой) запросили восстановление пароля от учётной записи на сайте 127.0.0.1:8000, которая связана с этим адресом электронной почты.

Пожалуйста, перейдите на эту страницу и введите новый пароль:
http://127.0.0.1:8000/auth/users/reset_password_confirm/MTQ/c167xm-46dd2e6e55ffeb0bdf4173d491116d8f
Your username, in case you've forgotten: alibaba@chudo.rf

Спасибо, что используете наш сайт!
Команда сайта 127.0.0.1:8000
```